modules/sd-planning-considerations.adoc

```diff
@@ -70,10 +70,10 @@ The maximum number of compute nodes on ROSA is 180.
 
 .Rules for infrastructure node resizing alerts
 
-Resizing alerts are triggered for the infrastructure nodes in a cluster when either of the following scenarios are true: 
+Resizing alerts are triggered for the infrastructure nodes in a cluster when the CPU or the memory has high sustained utilization. This high-sustained utilization status is: 
 
-* Each infrastructure node has less than 16GiB RAM or less than 5 CPUs, and there are more than 25 and less than 101 compute nodes.
-* Each infrastructure node has less than 32GiB RAM or less than 9 CPUs, and there are more than 100 compute nodes.
+* An infrastructure node sustains over 50% utilization in a classic ROSA cluster with a single availability zones using 2 infrastructure nodes.
```

Comment:
```suggestion
* Infrastructure nodes sustain over 50% utilization on average in a classic ROSA cluster with a single availability zones using 2 infrastructure nodes.
```

The alert triggers off of the total average (e.g. one node at 60% and one node at 30% wouldn't trigger a scaleup, but both nodes at 60% would) - open to other ways to word this too.